types/boom/index.d.ts

```diff
@@ -7,335 +7,332 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.4
 
-export = Boom;
-
-declare namespace Boom {
-
-    /**
-     * boom provides a set of utilities for returning HTTP errors. Each utility returns a Boom error response object (instance of Error) which includes the following properties:
-     * @see {@link https://github.com/hapijs/boom#boom}
-     */
-    export class Boom<Data = any> extends Error {
+declare module 'boom' {
```

Comment:
Please don't use `declare module`, instead export things directly from file-scope (like it was done before).

Reply:
Why exactly is this not good? What's the difference?

Reply:
`declare module "boom"` is an ambient module declaration -- it's a global definition and not bound to the particular file it appears in. This can cause hard-to-debug errors in projects with multiple `node_modules`, because if we see two of these there's a conflict (that would otherwise be resolved by the modules having different paths).

Reply:
@TimonVS there's [some info here](https://www.typescriptlang.org/docs/handbook/namespaces-and-modules.html) but I remember getting very confused by modules, namespaces, ambient modules, etc. so I've half learnt something new here but not grok'd it yet (I only copied what was there before).